src/ui/component/tooltip/tooltip.tsx

```diff
@@ -0,0 +1,57 @@
+import { useEffect, useState } from 'react'
+import type { FC } from 'react'
+import classNames from 'classnames'
+import * as RTooltip from '@radix-ui/react-tooltip'
+import './tooltip.scss'
+
+type TooltipProps = {
+  content: JSX.Element
+  trigger: JSX.Element
+  contentClassName?: string
+  triggerClassName?: string
+  side?: 'left' | 'right' | 'top' | 'bottom'
+  sideOffset?: number
+  container?: HTMLElement
```

Comment:
If I'm correct it's used for portals, maybe it could be expressed in the prop name to precise it

Reply:
Sure! Property name has been replaced with `portalContainer`.